### DISCUSSION TITLE
A mini Change For The Haywire

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2784] [Ideas]
A small change I thought of while I was watching YouTube Shorts. This change means that when Haywire chases Tux, his lighter above generates fire particles. This idea was shown in an official Rusty Box concept on the Supertux discord where all the sprites of all the bombs in the game were, and one of the Haywire's poses (exactly the one running towards Tux), had a bit of fire in its lighter.

Here A Image

![haywire](https://github.com/SuperTux/supertux/assets/139823928/95380820-c6cc-413d-9d49-99ce788bffae)

**Comment by komaros1** [comment]
I think that it would be strange to see this.

**Comment by Rusty-Box** [comment]
Overall this idea is great and is something I've been contemplating for awhile. Triggered bombs could emit smoke particles from their fuse area - potentially even adding a small flame as a particle effect on top.
